db-developments/bash/03_aggregate.sh

```diff
@@ -0,0 +1,16 @@
+source bash/config.sh
+
+display "Creating yearly aggregate tables"
+python3 python/aggregate.py sql/aggregate/yearly.sql 2010
+python3 python/aggregate.py sql/aggregate/yearly.sql 2020
+
+display "Creating spatial aggregate tables"
+#python3 python/aggregate.py sql/aggregate/spatial.sql 2010 block
+#python3 python/aggregate.py sql/aggregate/spatial.sql 2010 tract
+python3 python/aggregate.py sql/aggregate/spatial.sql 2020 block
+python3 python/aggregate.py sql/aggregate/spatial.sql 2020 tract
+python3 python/aggregate.py sql/aggregate/spatial.sql 2010 commntydst
+python3 python/aggregate.py sql/aggregate/spatial.sql 2010 councildst
+#python3 python/aggregate.py sql/aggregate/spatial.sql 2010 nta
+python3 python/aggregate.py sql/aggregate/spatial.sql 2020 nta
+python3 python/aggregate.py sql/aggregate/spatial.sql 2020 cdta
```

Comment:
This was previously a function in db-developments/devdb.sh

Reply:
wanna delete the commented out lines or keep and note as deprecated?

Reply:
They are deleted now, i originally commented before they were

Reply:
I think they're still there? [link to the file on this branch](https://github.com/NYCPlanning/data-engineering/blob/2023-06-fvk-devdb-gis-enhancements/db-developments/bash/03_aggregate.sh)

Reply:
Oh oops - I removed them on my sequel branch. Leaving them for now then. Sorry!

Reply:
Nevermind, just did it in this branch